### PR TITLE
add -y|-yes and -s flags

### PR DESCRIPTION
**Pull request by Yu1986**
## Description

add -y | -yes so will choose the default answers, skip prompting questions
add -s will skip the install 
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/commands/app/gen.js [hunk 1]
```diff
@@ -17,11 +18,22 @@ const { flags } = require('@oclif/command')
...
 class Gen extends BaseCommand {
   async run () {
-    // const { args } = this.parse(Gen)
+    const { args, flags } = this.parse(Gen)
     debug('creating new app with gen command')
+    let skip_prompt = false
+    if (flags.yes) {
+      skip_prompt = true
+    }
+    let skip_install = false
```
Comment (purplecabbage) [inline src/commands/app/gen.js]: This is a new flag, not a requirement.

File: src/commands/app/gen.js [hunk 2]
```diff
@@ -17,11 +18,22 @@ const { flags } = require('@oclif/command')
...
+    if (flags.skill_install) {
```
Comment (purplecabbage) [inline src/commands/app/gen.js]: skill should be skip, but ultimately this is un-needed code

File: src/generators/createGenerator.js [hunk 3]
```diff
@@ -28,6 +35,11 @@ will contain static assets to be uploaded to cloud storage. If you
 have a build process use your build's output directory.
 What folder do you want to use as your public web assets directory?
 `
+    let showPrompt = true
+    if (this.options.skip_prompt) {
+      showPrompt = false
+    }
+
```
Comment (purplecabbage) [inline src/generators/createGenerator.js]: `const showPrompt = !this.options.skip_prompt`

File: src/generators/createGenerator.js [hunk 4]
```diff
@@ -115,6 +140,9 @@ What folder do you want to use as your public web assets directory?
   }
 
   async install () {
+    if (this.options.skip_install) {
+      return
+    }
```
Comment (purplecabbage) [inline src/generators/createGenerator.js]: We should only ever skip-install if the user replies No to the prompt
If the --yes flag is present, we don't prompt at all, so just install ...

Comment (Yu1986) [reply src/generators/createGenerator.js]: fixed

**Pull request merged**